bin/alluxio-mount.sh

```diff
@@ -139,6 +148,7 @@ function mac_hfs_provision_sectors() {
 function mount_ramfs_linux() {
   init_env $1
 
+  ALLUXIO_WORKER_MEMORY_SIZE=$(${BIN}/alluxio getConf alluxio.worker.memory.size)
```

Comment:
good catch, I've removed this line. 

Now that ALLUXIO_WORKER_MEMORY_SIZE is set in init_env, we don't need to update it anywhere else.